Azure-Samples/azure-search-java-samples
# Search indexer CLI

## Purpose

Port of existing servlet-based source code to command-line. Updated code to Java 11 (refactored to use Optional and HttpClient).

Note that I _did not_ preserve Carey's servlet code. Doing so required a considerably more complex project that I felt might make it harder for the "happy path" CLI to be as straightforward as possible. 

## Does this introduce a breaking change?
```
[X] Yes
[ ] No
```

## Pull Request Type

```
[X ] Feature
[X ] Code style update (formatting, local variables)
```

## How to Test
* Check and follow instructions in `search-java-indexer-demo/README.md`

@HeidiSteen @LisaLeib
